src/main/java/uk/gov/ons/census/caseapisvc/endpoint/CaseEndpoint.java

```diff
@@ -159,6 +156,13 @@ public TelephoneCaptureDTO getNewQidForTelephoneCapture(
     return cases.stream().map(c -> buildCaseContainerDTO(c, false)).collect(Collectors.toList());
   }
 
+  @GetMapping(value = "/case_details/{case_id}")
```

Comment:
Normal convention in URL's is dashes rather than underscores  
```suggestion
  @GetMapping(value = "/case-details/{case_id}")
```